python/tvm/relay/transform/transform.py

```diff
@@ -726,6 +726,18 @@ def PartitionGraph():
     return _ffi_api.PartitionGraph()
 
 
+def MangleGraph(mod_name):
```

Comment:
nit : I think this is name mangling the functions, right ? We might need a name that indicates it is name mangling the function names. What do others think ? 

cc : @areusch @Mousius